jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/XmlBPMNProcessDumper.java

```diff
@@ -772,8 +773,10 @@ public void visitConnection(Connection connection, StringBuilder xmlDump, int me
                             xmlDump.append("language=\"" + JAVA_LANGUAGE + "\" ");
                         } else if ("XPath".equals(constraint.getDialect())) {
                             xmlDump.append("language=\"" + XPATH_LANGUAGE + "\" ");
-                        }  else if ("JavaScript".equals(constraint.getDialect())) {
+                        } else if ("JavaScript".equals(constraint.getDialect())) {
                             xmlDump.append("language=\"" + JAVASCRIPT_LANGUAGE + "\" ");
+                        } else if ("feel".equals(constraint.getDialect())) {
```

Comment:
nitpicking, but is called `FEEL` allcaps

![image](https://user-images.githubusercontent.com/1699252/38737632-1f3a693e-3f30-11e8-8529-60b8bc9faa96.png)